examples/seismic/model.py

```diff
@@ -98,6 +149,89 @@ def demo_model(preset, **kwargs):
         return Model(vp=v, origin=origin, shape=v.shape,
                      spacing=spacing, nbpml=20)
 
+    elif preset.lower() in ['marmousitti', 'marmousi2dtti']:
```

Comment:
So this is for TTI and the other for Acoustic ?

Reply:
yes

Reply:
should we then not have two different models, one for acoustic and one for tti (to be put in their respective folders) ? The common parts could then be left in a common module

Reply:
Well technically TTI is acoustic...
But separating model would be not very helpful in my opinion, havibg a set of "physical" example model in the same place makes it eaiser to play around

Reply:
I'm inclined to disagree. However...

let's at least be more consistent with names. 
``constant`` --> ``constant_acoustic` `
``constanttti`` --> ``constant_tti``
and so on and so forth.
This will make the 1-to-1 relationship with the ``acoustic`` and ``tti`` problems more clear